tst/testinstall/grppc.tst

```diff
@@ -166,6 +166,12 @@ gap> RepresentativeAction(G,x,x^2)<>fail;
 true
 gap> RepresentativeAction(G,x,x^2);
 f1*f2
+gap> g := SmallGroup(243,27);
+<pc group of size 243 with 5 generators>
+gap> Omega(g,3,1);
```

Comment:
The way the output is printed could change, and it's not even clear to me that it is correct now. How about this instead (and similar below):
```suggestion
gap> AsSet(Omega(g,3,1)) = Set(Filtered(g, g -> IsOne(g^3)));
true
```

Reply:
I was just stupid there, sorry. I changed the test.